resources/Materials/TestSuite/stdlib/color_management/ocio_color_management.mtlx

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<materialx version="1.39" colorspace="lin_rec709">
+  <standard_surface name="SR_OCIO_Demo" type="surfaceshader">
+    <input name="base_color" type="color3" nodegraph="NG_OCIO_Demo" output="out_color" />
+  </standard_surface>
+  <surfacematerial name="OCIO_Demo" type="material">
+    <input name="surfaceshader" type="surfaceshader" nodename="SR_OCIO_Demo" />
+  </surfacematerial>
+  <nodegraph name="NG_OCIO_Demo">
+    <output name="out_color" type="color3" nodename="switch1" />
+    <switch name="switch1" type="color3">
+      <input name="in1" type="color3" nodename="image1" />
+      <input name="in2" type="color3" nodename="image2" />
+      <input name="in3" type="color3" nodename="image4" />
+      <input name="in4" type="color3" nodename="image3" />
+      <input name="in5" type="color3" nodename="image5" />
+      <input name="which" type="float" nodename="add1" />
+    </switch>
+    <image name="image1" type="color3">
+      <input name="file" type="filename" value="resources/Images/color_palette_srgb.exr" colorspace="sRGB - Texture" />
+    </image>
+    <image name="image2" type="color3">
+      <input name="file" type="filename" value="resources/Images/color_palette_lin_p3d65.exr" colorspace="sRGB Encoded P3-D65 - Texture" />
+    </image>
+    <image name="image3" type="color3">
+      <input name="file" type="filename" value="resources/Images/color_palette_g18_rec709_tx.exr" colorspace="Gamma 1.8 Rec.709 - Texture" />
+    </image>
+    <image name="image4" type="color3">
+      <input name="file" type="filename" value="resources/Images/color_palette_lin_ap0.exr" colorspace="lin_ap0" />
```

Comment:
Question about all the color space names here ~ although the color spaces here are all in the CIF suite, none of them use the CIF names. I assume that it is meant to trigger delegation to OCIO to resolve the color spaces, rather than invoking any of the built in transformations? That approach makes sense to me, but I had to study the diff a fair bit to be sure I had come to the right conclusion. I was thinking it might be good for maintenance purposes to add documentation to this file so that when people encounter it in the future they don't inadvertently "clean it up" by changing the names to canonical names and thus silently bypass actually testing the OCIO functionality?